docs/README.md

```diff
@@ -42,7 +42,7 @@ The following sections are Kubernetes specific. If you are a Kubernetes user, us
 ### Container Images
 |EFS CSI Driver Version     | Image                               |
 |---------------------------|-------------------------------------|
-|master branch              |amazon/aws-efs-csi-driver:latest     |
+|master branch              |amazon/aws-efs-csi-driver:master     |
```

Comment:
Out of scope of this PR, but I think we should discourage users from using the latest/"master" tag by not mentioning it anywhere in the README. If users find it by reading the code, that's OK. But that doesn't apply to the majority of the users.